index.js

```diff
@@ -5,6 +5,15 @@ const zlib = require('zlib')
 const pump = require('pump')
 const sts = require('string-to-stream')
 const mimedb = require('mime-db')
+const isStream = require('is-stream')
+const intoStream = require('into-stream')
+const peek = require('peek-stream')
+const through = require('through2')
```

Comment:
Sure can. The original unzip-if-zip module used through2, so I didn't change it. What about using [`minipass`](https://www.npmjs.com/package/minipass) instead of `PassThrough`, since this is a hot path? I think it should work, unless you can't pipe a readable that doesn't implement `unpipe()` to an http.ServerResponse.

Reply:
go for it!